tests/zfs-tests/tests/functional/bclone/TODO

```diff
@@ -0,0 +1,5 @@
+- If dedup enabled, block_cloning uses dedup.
+- block_cloning works only with the same encryption key
```

Comment:
Perhaps this one can be removed, `block_cloning_cross_enc_dataset.ksh` was added for this.